components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -61,14 +60,21 @@ export default function OrganizationSelector() {
         // collaborator can't access projects, members, usage and billing
         if (hasMemberPermission) {
             // Check both flags as one just controls if the menu item is present, the other if the page is accessible
-            if (repoConfigListAndDetail && showRepoConfigMenuItem) {
+            if (configurationsAndPrebuilds) {
                 linkEntries.push({
                     title: "Repositories",
                     customContent: <LinkEntry>Repositories</LinkEntry>,
                     active: false,
                     separator: false,
                     link: "/repositories",
                 });
+                linkEntries.push({
```

Comment:
[nit, not-relates] Telemetry can tell us most clicked menu, then we can adjust the ordering of menu